static/js/base/ga.js

```diff
@@ -34,7 +34,12 @@ function triggerEvent(category, from, to, label) {
 
 if (dataLayer) {
   window.addEventListener("click", function(e) {
-    let target = e.target.closest("a");
+    let target = e.target;
+    if (!target || !e.target.closest) {
```

Comment:
We have a polyfill for `.closest` so not sure what that would fail.

This seems more like a workaround to avoid sentry errors, but if we are fine with dropping some GA events silently, then it's probably fine.

Reply:
I was thinking the issue is actually the 'target' that's missing for some reason, though I'm not sure how, because the browsers it's happening on don't need the polyfill.

Reply:
If target was undefined error would probably say something like "cannot call `closest` of undefined" I guess.
Maybe target is defined but is not a proper DOM node for some reason? Or is a 'document' or something that doesn't have `closest`? Anyway...

Current fix will do the trick - errors will stop appearing, GA event is not called anyway if it errors. So if we are fine with that, it's good to go. Not sure if we want to spend time investigating more.

Reply:
I guess the click could be on the `document` if someone is clicking randomly on the page? Or a `.textNode`? I think if we see a significant drop in GA events we should investigate further.